libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.integration.spec.ts

```diff
@@ -0,0 +1,368 @@
+import { RouterTestingModule } from '@angular/router/testing';
```

Comment:
These tests has been seperated out from `modal-wrapper.component.spec.ts` as they're now integration tests with the new implementation.

Reply:
👍